sphinxlint/__init__.py

```diff
@@ -1,7 +1,9 @@
 """Sphinx linter."""
 
-__version__ = "0.6.8"
+import importlib.metadata
 
 from sphinxlint.sphinxlint import check_file, check_text
 
+__version__ = importlib.metadata.version(__name__)
```

Comment:
Because the directory name ("sphinxlint") is different to the project ("sphinx_lint"):
```suggestion
__version__ = importlib.metadata.version("sphinx_lint")
```